primerpedia.js

```diff
@@ -67,8 +72,7 @@ function apiRequest(queryString) {
         throw new Error("Argument Error on queryString");
     }
 
-    // Show animated loading spinner -- from https://commons.wikimedia.org/wiki/File:Chromiumthrobber.svg
-    contentDivElement.innerHTML = "<img src='img/loading.svg' alt='Loading...' id='loading-spinner'/>";
+    renderLoadingSpinner();
```

Comment:
Is it worth DRY-ing this code out when (1) it's a single line and (2) it isn't repeated in the first place?

Reply:
Short Answer: Yes :)
Long Answer:
Putting something into a function does not necessary require it to be a change made to reduce repetition.
In this case I moved the assignment into a dedicated function to make the intent of the line clear and expressive. Whatever this assignment does needs to result into rendering/displaying a loading spinner.

In all honesty - this line (even in its current form) extremely bugs me since it is a side effect of this function and should not be there. But it is the most sensible solution. Even better than this would be to move the HTML generation into `utils.js` or even better than that have a dedicated HTML node instead of reassigning the content each time.